File: it.unibo.cautiousExplorer/backup/supports2021/ActorMsgs.java
package it.unibo.supports2021;

public class ActorMsgs {
    final static public String startTimerId      = "startTimer";
    final static public String endTimerId        = "endTimer";
    final static public String startTimerMsg     = "{\"ID\":\"TIME\" }".replace("ID", startTimerId);
    final static public String endTimerMsg       = "{\"ID\":\"ok\" }".replace("ID", endTimerId);

}

File: it.unibo.kotlinSupports/app/src/main/kotlin/it/unibo/supports/ActorMsgs.kt
package it.unibo.supports

import it.unibo.supports.ActorMsgs

object ActorMsgs {
    const val startTimerId = "startTimer"
    const val endTimerId   = "endTimer"
    val startTimerMsg      = "{\"ID\":\"TIME\" }".replace("ID", startTimerId)
    val endTimerMsg        = "{\"ID\":\"ok\" }".replace("ID", endTimerId)
}